src/strings/en-us.json

```diff
@@ -1347,6 +1347,7 @@
     "Saturday": "Saturday",
     "Save": "Save",
     "SaveChanges": "Save changes",
+    "SavePassword": "Save Password",
```

Comment:
```suggestion
    "SavePassword": "Save password",
```

👀

Reply:
I believe most buttons in web use Pascal casing... "Reset Password" directly below this certainly does 😉

Reply:
I don't like it though 😅